apps/arweave/include/ar_mining.hrl

```diff
@@ -28,7 +28,8 @@
 	seed = not_set, %% serialized
 	session_key = not_set, %% serialized
 	start_interval_number = not_set, %% serialized
-	step_number = not_set %% serialized
+	step_number = not_set, %% serialized
+	label = <<"not_set">> %% not term, for prevent term table pollution DoS
```

Comment:
"term" here mean `atom`? Like you want to prevent a malicious actor from submitting a lot of different shares with different labels and overflowing the erlang 1,048,576 atom limit?